### PR TITLE
Moving the source from my private repo to Elastic's repo

### PR DESCRIPTION
**Pull request by alexh97**
Resolves: https://github.com/elastic/infra/issues/16406

### CODE REVIEW

File: project-assigner/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  	  issues:
+    	  types: [labeled, unlabeled]
+
+	jobs:
+	  assign_to_project:
+	    runs-on: ubuntu-latest
+	    name: Assign an issue to project based on label
+	    steps:
+	      - name: Assign to project
+		uses: alexh97/github-actions/project-assigner@v1.0.0
```
Comment (leathekd) [inline project-assigner/README.md]: Should replace `alexh97` with `elastic` here.

File: project-assigner/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	  pull_request:
+	    types: [labeled, unlabeled]
+
+	jobs:
+	  assign_to_project:
+	    runs-on: ubuntu-latest
+	    name: Assign a PR to project based on label
+	    steps:
+	      - name: Assign to project
+		uses: alexh97/github-actions/project-assigner@v1.0.0
```
Comment (leathekd) [inline project-assigner/README.md]: Should replace `alexh97` with `elastic` here.

**Comment by leathekd** [comment]
I can't see the diff for the `dist/index.js` file- does that need to be rebuilt now that it's in a different repo?

File: project-assigner/README.md (hunk 1 above)
Comment (alexh97) [reply project-assigner/README.md]: Ah, yes, good catch.

**Pull request merged**